cdc/log.cc

```diff
@@ -1518,6 +1518,18 @@ class transformer final : public change_processor {
         auto process_cell = [&, this] (const column_definition& cdef) {
             if (auto current = get_col_from_row_state(row_state, cdef)) {
                 _builder->set_value(image_ck, cdef, *current);
+            } else if (op == operation::pre_image) {
```

Comment:
I would extract the `if` outside the lambda to check it once not for every column. So auto process_cell = op == operation::pre_image ? ... : ...;

Reply:
I don't see a clean way to do it, without code duplication.